### PR TITLE
CAL-226 moved format-specific attributes to appendix.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Moves metadata attributes and catalog taxonomy to an appendix in the documentation.

#### Who is reviewing it?

@vinamartin @Lambeaux @jrnorth @ahoffer @brendan-hofmann 

#### Choose 2 committers to review/merge the PR.

@clockard 
@coyotesqrl
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested?

review for outline/organization. New content will come from future PRs.

#### What are the relevant tickets?

[CAL-226](https://codice.atlassian.net/browse/CAL-226)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/alliance/files/1089407/documentation.pdf)

#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 1]
```diff
@@ -1,14 +1,14 @@
 
-====== Attributes Provided by ${cal-branding}
+=== Attributes Provided by ${cal-branding}
 
-${cal-branding} includes support for accessing additional metadata from ISR specific data formats. 
+${cal-branding} includes support for accessing additional metadata from ISR specific data formats.
```
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: ❓ Should 'ISR specific' be hyphenated?

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 .${cal-branding} supported data formats:
 * MPEG2/4 Video Transport Streams including MISB compliant KLV data
-* Image data formats - NITF 2.0,2.1 / NSIF (STANG 4545) 
+* Image data formats - NITF 2.0,2.1 / NSIF (STANG 4545)
```
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: ❓ Should 'STANG' be STANAG' here?

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**